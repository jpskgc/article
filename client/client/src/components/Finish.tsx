import React from 'react';
import {Container, Message} from 'semantic-ui-react';

class Finish extends React.Component<{}, {}> {
  render() {
    return (
      <Container text style={{marginTop: '3em'}}>
        <Message color="green">Your post was successful</Message>
      </Container>
    );
  }
}

export default Finish;
