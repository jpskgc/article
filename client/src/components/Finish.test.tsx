import React from 'react';
import Finish from './Finish';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Finish />).toJSON();
  expect(tree).toMatchSnapshot();
});
