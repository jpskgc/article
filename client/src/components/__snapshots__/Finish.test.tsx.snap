// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ui text container"
  style={
    Object {
      "marginTop": "3em",
    }
  }
>
  <div
    className="ui green message"
  >
    Your post was successful
  </div>
  <a
    className="ui green button"
    href="/"
    onClick={[Function]}
    role="button"
  >
    <i
      aria-hidden="true"
      className="arrow left icon"
      onClick={[Function]}
    />
    Back to Home
  </a>
</div>
`;
